{"name": "test_get_orders_with_auth", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "self = <test_get_orders.TestGetOrders object at 0x102e896a0>\nauth_header = {'Authorization': 'Bearer <accessToken>'}\n\n    @allure.feature(\"Получение заказов\")\n    @allure.story(\"Получение заказов авторизованного пользователя\")\n    def test_get_orders_with_auth(self, auth_header):\n        response = requests.get(f\"{BASE_URL}/orders\", headers=auth_header)\n>       assert response.status_code == 200  # Ожидаем успешное получение заказов\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\ntest_get_orders.py:18: AssertionError"}, "start": 1732753657585, "stop": 1732753657952, "uuid": "0562492c-cad0-45ba-88f1-974445f872a6", "historyId": "2fab9c5df7879b7e1f6bf1dc568f2457", "testCaseId": "2fab9c5df7879b7e1f6bf1dc568f2457", "fullName": "tests.test_get_orders.TestGetOrders#test_get_orders_with_auth", "labels": [{"name": "story", "value": "Получение заказов авторизованного пользователя"}, {"name": "feature", "value": "Получение заказов"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "23686-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_orders"}]}